src/hardware/sblaster.cpp

```diff
@@ -294,6 +308,136 @@ static void InitializeSpeakerState()
 	}
 }
 
+
+static const std::map<SB_TYPES, FILTER_TYPES> sb_type_to_filter_type_map = {
+        {SBT_NONE, FT_NONE},
+        {SBT_1, FT_SB1},
+        {SBT_2, FT_SB2},
+        {SBT_PRO1, FT_SBPRO1},
+        {SBT_PRO2, FT_SBPRO2},
+        {SBT_16, FT_SB16},
+        {SBT_GB, FT_NONE},
+};
+
+static const std::map<std::string, FILTER_TYPES> filter_map = {
+        {"none", FT_NONE},
+        {"sb1", FT_SB1},
+        {"sb2", FT_SB2},
+        {"sbpro1", FT_SBPRO1},
+        {"sbpro2", FT_SBPRO2},
+        {"sb16", FT_SB16},
+};
+
+static void configure_filters(Section_prop* config)
+{
+	auto set_filter_params = [](const std::string conf,
+	                            FILTER_TYPES &filter_type_out,
+	                            bool &filter_force_out) {
+
+		const auto filter = split(conf);
```

Comment:
At this point, the `filter` variable is holding a vector of user preferences - which might be empty or a bunch of garbage.  Suggest a calling it something closer to reflect "user data".

In some of my code, I postfix this with `_pref`, indicating that it's a user preference and still needs digesting. 

(very minor)

Reply:
Yeah, and I've renamed this to `tokens` in a later commit. If `*_prefs` is a common pattern, I'll rename it to that.